### DISCUSSION TITLE
Questions about the new Fixed Version Runtime v120 or above constraint

### DISCUSSION THREAD

**Discussion by btlopener** [MicrosoftEdge/WebView2Feedback#4239] [Q&A] [answered]
In the latest release, there is a breaking change that states: "_If you're developing an unpackaged Win32 app using Fixed Version Runtime v120 or above and targeting Windows 10 devices, you need to run a couple of ACL shell commands (icacls) to avoid crashes_".

**Under what circumstances can these crashes occur?**

We have many customers who are very particular about changing permissions and security. By default, these permissions are already set (in the Programs Files folder), but this is not guaranteed. 

**Where can I find official documentation stating that this change is safe and won't introduce any security issues?**

Many thanks!

**Comment by liminzhu** [comment, marked as answer]
Thanks for opening this @btlopener ! The crash would happen at launch if they were to happen. 

To expand on this more, renderer process is now being run in AppContainer, and icalcs command listed in the [documentation](https://learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/distribution?branch=main&branchFallbackFrom=pr-en-us-2961#details-about-the-fixed-version-runtime-distribution-mode) allows for the fixed version binaries to be accessible in this context. With evergreen WebView2, the runtime's installation/deployment process takes care of that, but for fixed version, they are just loose binaries carried by the developer so that extra step to grant permission needs to be taken by the developer. This could also be potentially handled by your app's packaging process (which for example is why this is not a problem in MSIX), but if they are not, WebView2 should deterministically crash on launch since it cannot successfully create renderer process. The renderer in app container update is documented in Edge's [release notes](https://learn.microsoft.com/en-us/deployedge/microsoft-edge-relnote-stable-channel#version-1200221061-december-7-2023).

**Reply by jebihug** [reply]
I am now using fixed runtime v120 with my application and I have not applied the acl commands and it still works.
